### ISSUE TITLE
"this github" link no longer working

### ISSUE THREAD

**Issue by ath0rus** [abcminiuser/python-elgato-streamdeck#119]
The "this github" link no longer works due to the repository being privated (or take down). This is where the link in question can be found: [Link](https://github.com/abcminiuser/python-elgato-streamdeck/tree/master#credits)

**Comment by abcminiuser** [comment]
Hrm, that's odd. All the author's old repos look to be deleted. I did find an old revision of the file in question in https://github.com/Julusian/node-elgato-stream-deck/commit/8525bee76ccda356e9c34f208792bd3f7eba83f8#diff-99745d225ce143b02ff42aa535dd47e4e0bdf87c7a4aec1578415704e3a329c4 but I'm not really sure what happened to the project.

**Comment by abcminiuser** [comment]
Ah, looks like https://github.com/alvancamp/node-elgato-stream-deck/blob/master/NOTES.md is the "new" correct link.

**Comment by abcminiuser** [comment]
Fixed in 87f4f0fb24272f3d3ec51cdc446a4aeacafe6dce

**Comment by Julusian** [comment]
A bit of history, is the original author deleted their github account (including the repository). as I was the other with permission to publish the package to npm (https://www.npmjs.com/package/elgato-stream-deck) I restored what I could (minus issues) at my own repository https://github.com/Julusian/node-elgato-stream-deck, and continued work (under the original name on npm).
Some time later, they restored their github account under a new name, as someone else had now taken their original username.

In my repository, I did remove that notes file a while ago, as it only covers the original revision of the original model
